src/Integration.Vsix/source.extension.vsixmanifest

```diff
@@ -1,17 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="b039a29c-3109-44e1-9695-cededc24dfec" Version="0.9" Language="en-US" Publisher="SonarSource SA and Microsoft Corporation" />
+    <Identity Id="SonarLint.8c442ec8-0208-4840-b83f-acd24f41acf7" Version="2.0.0" Language="en-US" Publisher="SonarSource SA and Microsoft Corporation" />
     <DisplayName>SonarLint for Visual Studio</DisplayName>
-    <Description xml:space="preserve">Connect Visual Studio and SonarQube to enforce code development rules and comply with governance policies, and to minimize your technical debt</Description>
-    <MoreInfo>https://github.com/SonarSource-VisualStudio/sonarlint-visualstudio</MoreInfo>
+    <Description xml:space="preserve">Provides on-the-fly feedback to developers on new bugs and quality issues injected into C# and VB.Net code. Further, it allows connecting Visual Studio and SonarQube to enforce shared code development rules and comply with governance policies, and to minimize your technical debt.</Description>
```

Comment:
Here's a slightly different version @tvsonar:
Provides on-the-fly feedback to developers on new bugs and quality issues in C# and VB.Net code to minimize technical debt. It can also optionally connect Visual Studio to SonarQube to enforce shared development rules and compliance with governance policies.